gattc_windows.go

```diff
@@ -365,7 +365,8 @@ func (c *DeviceCharacteristic) Read(data []byte) (int, error) {
 // notification with a new value every time the value of the characteristic
 // changes.
 func (c DeviceCharacteristic) EnableNotifications(callback func(buf []byte)) error {
```

Comment:
I suggest doing this in a new `EnableIndications` instead of reusing `EnableNotifications`. To avoid code duplication, you can refactor most of the body into a separate function.